src/media/js/site/actions/ui.js

```diff
@@ -0,0 +1,11 @@
+import {createAction} from 'redux-actions';
```

Comment:
how come we need to pull the state into the application level? it should suffice to have the state be part of the component since the header is never going to detach